.gitignore

```diff
@@ -15,3 +15,4 @@ report
 jacoco.data
 .gradle
 build
+*.bai
```

Comment:
Yeah, a couple popped out when I ran tests. Should I move this to my global ignore file instead?